### PR TITLE
[QPE-230] First take on writing version to qext-file

### PR DESCRIPTION
**Pull request by marols**
Writing version to qext-file

### CODE REVIEW

**Comment by marols** [comment]
@flennmark you asked for a version in each zip-file to know which version you were currently working on. The qext file was already prepared to hold the version. Will this solution do? To see the version:

1. Open the artifact in CircleCI
2. Unpack it
3. Find the version property inside the qext-file

**Comment by marols** [comment]
I have now also added the version to the zip-file name. It took some refactoring of the build script but it feels nice to have the version there as well.

@kristofferlind any thoughts on my implementation and changes?

**Comment by marols** [comment]
Damn, just realised I need a different way to version on other branches than master

**Comment by marols** [comment]
Okay, so I think I figured everything out now. For `master` the version will be a semantic version just as before and the artefact will be named like `qlik-heatmap-chart_0.32.0.zip`.

For all other branches, the version will be a combination of the branch name and the build number, like `qlik-heatmap-chart_[feature-QPE-230]-238.zip`

File: gulpfile.js [hunk 1]
```diff
@@ -41,8 +43,16 @@ gulp.task('webpack-build', done => {
   });
 });
 
+gulp.task('update-qext-version', function () {
+  return gulp.src("./build/" + name + ".qext")
```
Comment (kristofferlind) [inline gulpfile.js]: need to use buildDestination from settings, otherwise it won't work when BUILD_PATH is set

Comment (marols) [reply gulpfile.js]: I'm on it!

**Comment by marols** [comment]
@kristofferlind please review my changes

**Pull request merged**